R-Interface/R/symlinkTools.R

```diff
@@ -65,33 +65,36 @@ determineOverlap <- function(targetRoot, sourceRoot)
   return(list(targetToSource=targetToSource, sourceToTarget=sourceToTarget))
 }
 
-#Use overlapfunctions as returned by determineOverlap to generate a function to turn target-path from absolute to relative
+# Use overlapfunctions as returned by determineOverlap to generate a function to turn target-path from absolute to relative
 getRelativityFunction <- function(modulesRoot, renvCache)
 {
-  #I wanted this code to be more general but then it is too complicated to debug. And has a bug. And renv-cache can be assumed to be right next to Modules anyway...
+  # I wanted this code to be more general but then it is too complicated to debug. 
+  # And has a bug. And renv-cache can be assumed to be right next to Modules anyway...
```

Comment:
So, what I did in a few lines below the comment isn't necessary anymore, even now that some stuff has moved around?

Reply:
`And renv-cache can be assumed to be right next to Modules anyway...` isnt true anymore right?

And all the `determineOverlap` isnt necessary anymore if we always know that it is `Modules/renv-cache` right?

Anyway, im not too proud of this R file, I did wrote it but it got out of hand because I was trying to be overly generic.
If it works (which it seems to do) I am totally ok with these changes ;)

Reply:
> isn't true anymore right?

No, it's now lives inside the `Modules/renv-cache`.

It works, and I think we can leave it until maybe at some point we do this using CMake or PowerShell.